src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -1306,7 +1306,7 @@ private CallResult ExecuteCall<TTracingInstructions>(EvmState vmState, ReadOnlyM
                             }
                         }
 
-                        InstructionExtCodeSize(address, ref stack, spec);
+                        InstructionExtCodeSize(address, ref stack, txCtx.AuthorizedCode, spec);
```

Comment:
Still in some places we go with a standard `codeInfoRepository`, but sometimes go to `txCtx`, can we unify it? And I think better is to go with the field as it was done before, just set that field to AuthorizationContext if needed?